specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/stable/2021-11-12/hanaonazure.json

```diff
@@ -0,0 +1,707 @@
+            "Updating",
+            "Failed",
+            "Succeeded",
+            "Deleting",
+            "Migrating"
+          ],
+          "x-ms-enum": {
+            "name": "HanaProvisioningStatesEnum",
+            "modelAsString": true
+          },
+          "description": "State of provisioning of the HanaInstance"
+        }
+      },
+      "description": "Describes the properties of a HANA instance."
+    },
+    "HardwareProfile": {
+      "properties": {
+        "hardwareType": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Cisco_UCS",
+            "HPE"
+          ],
+          "x-ms-enum": {
+            "name": "HanaHardwareTypeNamesEnum",
+            "modelAsString": true
+          },
+          "description": "Name of the hardware type (vendor and/or their product name)"
+        },
+        "hanaInstanceSize": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "S72m",
+            "S144m",
+            "S72",
+            "S144",
+            "S192",
+            "S192m",
+            "S192xm",
+            "S96",
+            "S112",
+            "S224",
+            "S224m",
+            "S224om",
+            "S224oo",
+            "S224oom",
+            "S224ooo",
+            "S384",
+            "S384m",
+            "S384xm",
+            "S384xxm",
+            "S448",
+            "S448m",
+            "S448om",
+            "S448oo",
+            "S448oom",
+            "S448ooo",
+            "S576m",
+            "S576xm",
+            "S672",
+            "S672m",
+            "S672om",
+            "S672oo",
+            "S672oom",
+            "S672ooo",
+            "S768",
+            "S768m",
+            "S768xm",
+            "S896",
+            "S896m",
+            "S896om",
+            "S896oo",
+            "S896oom",
+            "S896ooo",
+            "S960m"
+          ],
+          "x-ms-enum": {
+            "name": "HanaInstanceSizeNamesEnum",
+            "modelAsString": true
+          },
+          "description": "Specifies the HANA instance SKU."
+        }
+      },
+      "description": "Specifies the hardware settings for the HANA instance."
+    },
+    "Disk": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The disk name."
+        },
+        "diskSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Specifies the size of an empty data disk in gigabytes."
+        },
+        "lun": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Specifies the logical unit number of the data disk. This value is used to identify data disks within the VM and therefore must be unique for each data disk attached to a VM."
+        }
+      },
+      "description": "Specifies the disk information for the HANA instance"
+    },
```

Comment:
The diff b/w this file and json file for preview version shows some unexpected deletions from this line forward related to sap system id, why is that so?

Reply:
some of the sap monitor related definitions are not required as we are not supporting the monitor commands.

Reply:
required apis added back